### DISCUSSION TITLE
How do I get my Instance on another Instance's list? (Such a Universodon).

### DISCUSSION THREAD

**Discussion by auwsom** [mastodon/mastodon#21441] [Q&A] [answered]
I tried searching for this answer, but have not found it yet.

**Comment by NicolaeNMV** [comment, marked as answer]
What do you exactly mean by that "on another Instance's list" ? Which list ?
You can already message anyone from any mastodon instances. You don't have to get into a list.

**Reply by auwsom** [reply]
Thanks.. if I host my own instance. Doesn't Universodon have a lost of instances it reposts automatically?

**Reply by nightpool** [reply]
I don't know what Universodon is, but generally no, that's not how Mastodon works.

**Reply by NicolaeNMV** [reply]
No problem. The servers talk to each other as equal to send and receive messages of the user that follow each other.

**Reply by auwsom** [reply]
@NicolaeNMV wait, I know there is blocking on an instance level, so I thought broadcasting was similar. I thought I had seen somewhere that Universodon didn't include some instances like masto.social.

Wait, so does that mean there are no recommendations algo?

**Reply by auwsom** [reply]
I don't think this is totally correct. I had a friend in Discord tell me: 'When you launch a new instance, it's not federated at all.  For an example, lets call your new instance A.  To begin with, the federated timeline on A will be empty except for local posts.  When a user on A follows someone on instance B for the first time, A federates with B causing public posts on B to show up in the federated timeline of instance A."

So it sounds like to have B show my public posts on A, I need to have one user on B like a post on my instance A. Can anyone confirm this?

I'm still interested in how recommendation algo works if there is one. Thanks

**Reply by auwsom** [reply]
"There's no algorithm recommending content to you, so you need to start following people before you can see posts."
https://www.wired.com/story/how-to-get-started-use-mastodon/

Recommending Users:Whom to Follow on Federated Social Networks
https://arxiv.org/pdf/1811.09292
